include/prism/regexp.h

```diff
@@ -37,6 +39,7 @@ typedef void (*pm_regexp_error_callback_t)(const uint8_t *start, const uint8_t *
  * @param name_data The optional data to pass to the name callback.
  * @param error_callback The callback to call when a parse error is found.
  * @param error_data The data to pass to the error callback.
+ * \public \memberof pm_parser
```

Comment:
I don't know about this one. It's only public so we can fuzz it, and while I understand that it's got a parser as the first parameter, it's not really meant as a member function, it's more a joining function between two modules.